src/js/components/TaskTable.js

```diff
@@ -165,6 +184,12 @@ class TaskTable extends React.Component {
     );
   }
 
+  renderHost(prop, task) {
+    let marathonTask = MarathonStore.getTaskFromTaskID(task.id);
```

Comment:
Maybe we should implement some sort of caching in `MarathonStore#getTaskFromTaskID`? Is that crazy talk?